pkg/file/pipeline/builder/builder_test.go

```diff
@@ -71,7 +71,7 @@ func TestEmpty(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	exp := swarm.MustParseHexAddress("ffd70157e48063fc33c97a050f7f640233bf646cc98d9524c6b92bcf3ab56f83")
+	exp := swarm.MustParseHexAddress("b34ca8c22b9e982354f9c7f50b470d66db428d880c8a904d5fe4ec9713171526")
```

Comment:
whoops... how is this possible that the zero hash changed now?

Reply:
it was incorrect what can i do :) ?

Reply:
WDYM incorrect? Both the reference hasher and the old bmt hasher give out a different hash for this stuff. What changed in this PR that changes this hash? and will the old hash still be correct?